lec12-datasets.Rmd

```diff
@@ -0,0 +1,166 @@
+---
+title: "Introduction to the datasets, scientific method, and team dynamics; Start the projects"
+author: "Luke Johnston"
+---
+
+## Lesson preamble:
+
+> ### Lesson objectives:
+> 
+> - To start the process of identifying which dataset to use and some possible research questions
+> - Learn about the scientific method and (the basics of) how to apply it to your project
+> - Learn basics of group dynamics in a team setting, how to share responsibilities and duties
+> - Get into groups and start your projects
+> 
+> ### Lesson outline:
+> 
+> Total lesson time: 2 hours
+> 
+> - Datasets to use (10-20 min)
+> - The scientific method (20 min)
+> - Team dynamics and assigning roles and tasks (15-20 min)
+> - Break (10 min)
+> - Getting into groups (10 min)
+>     - Exercise (25-30 min)
+>     - Start project work (20-25 min)
+
+-----
+
+## Datasets
+
+### List of available datasets
+
+{{list/link them here}}
+
+## The scientific method
+
+<!-- What is the scientific method? -->
+
+### Simplified diagram of the scientific method
+
+```{r fig_scientific_method, echo=FALSE}
+DiagrammeR::grViz('
+digraph rmarkdown {
+    graph [layout = neato, splines = true]
+    node [shape = box, style = rounded]
+    Hypothesis [pos = "0.75,3.75!"]
+    CollectData [label = "Collect Data", pos = "0.75,3!"]
+    AnalyzeData [label = "Analyze Data", pos = "0.75,2.25!"]
+    HypothesisTrue [label = "Hypothesis\nif TRUE", pos = "0,1.5!"]
+    HypothesisFalse [label = "Hypothesis\nif FALSE", pos = "1.5,1.5!"]
+    Report [label = "Disseminate", pos = "0.75,0.75!"]
+
+    Hypothesis -> CollectData -> AnalyzeData -> HypothesisTrue -> Report
+    AnalyzeData -> HypothesisFalse -> Report
+    {CollectData AnalyzeData Report HypothesisFalse} -> Hypothesis
+}')
+```
+
+### Steps in the scientific process[^sci_method]
+
+1. Identify research question(s).
+2. Look into what the previous literature shows. <!--(how much emphasis on this?)-->
```

Comment:
I think this is always important tbh, it's part of any scientific write-up to give an overview of where the literature is at. But we don't really have a way to grade this until the final project...?